Jenkinsfile.promote

```
@@ -83,11 +94,44 @@ pipeline {
                 }
             }
         }
-    }
-    post {
-        always {
-            script {
-                cleanImages()
+        stage('Merge PR')
+        {
+            when {
+                expression {
+                    return isRelease() && params.DEPLOY_BUILD_URL != ''
```

Once we read the deploy properties, we do not care anymore about `DEPLOY_BUILD_URL`. 
Instead, we should check the pr link is existing.

This should be:

```suggestion
                    return isRelease() && getDeployProperty('kogito-images.pr.link') != ''
```